### PR TITLE
:arrow_up: nightly rust

### PR DESCRIPTION
**Pull request by Undin**

### CODE REVIEW

File: pretty_printers_tests/src/test_runner.rs [hunk 1]
```diff
@@ -290,6 +312,34 @@ fn parse_debugger_commands(src_path: &Path, debugger_prefixes: &[&str]) -> Resul
                     line.as_str()
                 };
 
+                match parse_rustc_version(line, "min-version:") {
+                    Ok(Some(min_version)) => {
+                        if rustc_version < min_version {
+                            return DebuggerCommands::Skip(
+                                format!("Current rustc version ({}) less than minimum version ({}) required for test", rustc_version, min_version)
```
Comment (ortem) [inline pretty_printers_tests/src/test_runner.rs]: **is** less than

File: pretty_printers_tests/src/test_runner.rs [hunk 2]
```diff
@@ -290,6 +312,34 @@ fn parse_debugger_commands(src_path: &Path, debugger_prefixes: &[&str]) -> Resul
...
+                        return DebuggerCommands::Err(format!("Error while parsing rustc version: {}", e))
+                    },
+                    _ => {}
+                }
+
+                match parse_rustc_version(line, "max-version:") {
+                    Ok(Some(max_version)) => {
+                        if rustc_version > max_version {
+                            return DebuggerCommands::Skip(
+                                format!("Current rustc version ({}) greater than maximum version ({}) required for test", rustc_version, max_version)
```
Comment (ortem) [inline pretty_printers_tests/src/test_runner.rs]: **is** greater than

**Comment by Undin** [comment]
@ortem fixed

**Comment by Undin** [comment]
bors retry

**Comment by Undin** [comment]
bors r=ortem

**Pull request merged**